index.js

```diff
@@ -22,8 +22,9 @@ module.exports = (app) => {
         text: 'We check if any task lists need completing before you can merge this PR'
       }
     };
+
     // all finished?
-    if (! hasOutstandingTasks) {
+    if (hasOutstandingTasks !== null) {
```

Comment:
I think hasOutstandingTasks will be null if completed rather than not null here.

Reply:
Right you are! I forgot to change that after switching to regular expressions. I’ll push a correction shortly.